### DISCUSSION TITLE
Version 8.2 Can no longer download the m4a from Smule.com

### DISCUSSION THREAD

**Discussion by ErikVSB** [aclap-dev/video-downloadhelper#191] [Help and Q&A]
### Your question

With the 8.2 Update, Smule.com no longer allows for audio download in Firefox nor Chrome. Videos appear to still work but with Version 8.1.4.0, the download list would include an m4a link if the page was just audio. Any way to get that functionality back?

### Addon & CoApp version and details

**WORKED:**
Video DownloadHelper
Version 8.1.4.0
Browser locale: en-US
Production build
Built on 2024-01-04
Build options: browser=firefox, linuxlic=false, noyt=false

**NO LONGER WORKING:** 
Video DownloadHelper
Version 8.2.0.23 (stable)
Browser locale: en-US
Production build
Built on 2024-02-14
Build options: browser=firefox, linuxlic=false, noyt=false, target=mozilla

**Comment by paulrouget** [comment]
I can reproduce. I'll look into it!

Thank you.

**Comment by paulrouget** [comment]
Found out what the issue is. Version 8.2.1.3 will fix this issue. Hope to have a release in the coming days.

**Reply by ErikVSB** [reply]
A quicker response than was necessary but appreciated all the same. Thank you for all you do and all you're doing. Good luck with the updates.
